### DISCUSSION TITLE
how to change TRUSTED_DOMAINS after install?

### DISCUSSION THREAD

**Discussion by bobloadmire** [nextcloud/all-in-one#2623] [Questions] [answered]
I found the config.php in the apache container with the trusted domains entry.

1. how do I edit it? terminal says NANO is invalid.
2. is there anything else I need to edit?

**Comment by Zoey2936** [comment, marked as answer]
Do you want to support multiple domains for your instance? If yes, that will not work and is not supported

**Reply by bobloadmire** [reply]
No, I'm asking how to change the one I have.

**Reply by Zoey2936** [reply]
Please see here: https://github.com/nextcloud/all-in-one#how-to-change-the-domain
